recipes/pycircstat/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pycircstat" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 76c6c6a0d9b03e50c379deb0dc3d62781e5cfc0c5b45751ce13f6b3783049f34
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
+    - scipy
+    - pandas
+    - decorator
+    - sphinx
+    - sphinx_rtd_theme
+    - nose
+    - mock
```

Comment:
This is not a mistake – unfortunately, the package is designed such that it needs all runtime dependencies already installed when running `pip install`.

Reply:
Thanks for posting the explanation before we asked. Saves us a few hours debugging this :smile: